erigon-lib/state/domain_test.go

```diff
@@ -2432,6 +2436,7 @@ func TestDomain_PruneSimple(t *testing.T) {
 
 func TestDomainContext_findShortenedKey(t *testing.T) {
 	db, d := testDbAndDomain(t, log.New())
+	defer d.Close()
```

Comment:
`testDbAndDomain` inside does `t.Cleanup(db.Close)` - this is reason why no close outside